### PR TITLE
JBPM-8153 Container is removed from UI even though it was not possibl…

### PR DESCRIPTION
**Pull request by bxf12315**
…e to stop it.

**Thank you for submitting this pull request**

**JIRA**: [JBPM-8153](https://issues.redhat.com/browse/JBPM-8153)

**Referenced Pull Requests**:
* https://github.com/kiegroup/droolsjbpm-integration/pull/2358
* link 2
* link 3 etc.

**Business Central**: [WAR file](https://donwnload.here/something)

**VS Code**: [plugin](https://donwnload.here/something)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* Retest PR: <b>jenkins retest this</b>
* A full downstream build: <b>jenkins do fdb</b>
* A compile downstream build: <b>jenkins do cdb</b>
* A full production downstream build: <b>jenkins do product fdb</b>
* An upstream build: <b>jenkins do upstream</b>
</details>

### CODE REVIEW

**Comment by bxf12315** [comment]
dependent on https://github.com/kiegroup/droolsjbpm-integration/pull/2358

**Comment by sutaakar** [comment]
jenkins do fdb

**Comment by sutaakar** [comment]
jenkins do fdb

**Comment by sutaakar** [comment]
jenkins do fdb

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-api/src/main/java/org/kie/workbench/common/screens/server/management/service/ContainerService.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+package org.kie.workbench.common.screens.server.management.service;
+
+import org.jboss.errai.bus.server.annotations.Remote;
+import org.kie.server.controller.api.model.spec.ContainerSpec;
+
+@Remote
+public interface ContainerService {
+
+    boolean isRunningContainer(ContainerSpec containerSpec);
```
Comment (sutaakar) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-api/src/main/java/org/kie/workbench/common/screens/server/management/service/ContainerService.java:24]: Can you please add a Javadoc describing the method?

Comment (bxf12315) [reply kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-api/src/main/java/org/kie/workbench/common/screens/server/management/service/ContainerService.java:29]: Done

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by sutaakar** [comment]
@bxf12315 There is still a possibility to stop the container with active process instance. 
If container is in Deactivated state then pressing Stop button cause error message and Business central tries to stop the container.
Expected behaviour would be to show warning message (same as in active state) and prevent container stop action.

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by bxf12315** [comment]
d

> @bxf12315 There is still a possibility to stop the container with active process instance.
> If container is in Deactivated state then pressing Stop button cause error message and Business central tries to stop the container.
> Expected behaviour would be to show warning message (same as in active state) and prevent container stop action.

Fix it, please review again.

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by bxf12315** [comment]
jenkins retest this

**Comment by bxf12315** [comment]
jenkins do fdb

**Review by sutaakar — APPROVED** [review]

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by bxf12315** [comment]
jenkins retest this

**Review by elguardian — APPROVED** [review]
LGTM

**Review by nmirasch — APPROVED** [review]

**Pull request merged**